docker-compose.no-ssl.yml

```diff
@@ -1,15 +1,72 @@
 version: '2.1'
 services:
 
+    mender-useradm:
+        command: server --automigrate
+        volumes:
+            - ./keys/useradm/private.key:/etc/useradm/rsa/private.pem
+
+    mender-device-auth:
+        command: server --automigrate
+        volumes:
+            - ./keys/deviceauth/private.key:/etc/deviceauth/rsa/private.pem
+
+    mender-inventory:
+        command: server --automigrate
+
```

Comment:
I don't get why we need these settings here, can you please explain?

Reply:
I copied over these settings from `docker-compose.demo.yml`.

The problem here is that you cant run it like this:
```
docker-compose -f docker-compose.yml -f docker-compose.demo.yml -f docker-compose.no-ssl.yml up
```
because the demo file mounts the certificates for api gateway and storage proxy. Sadly in compose you cannot revoke this in a compose file which got merged later.

So I copied everything from the demo file and removed all the lines according to certificates.

With these changes you can run the command I mentioned in the README file and it works. Otherwise, the user needs to do changes on his own.

Reply:
The right way to run the demo is using the demo script, indeed.
`./demo -f docker-compose.no-ssl.yml up`
This brings a full working demo with HTTP-only support, see my other PR for details.

Reply:
Now I understand what you are trying to do :)  
That way the certificates still get mounted but they will be ignored.

Problem: the storage proxy still got HTTPS. I will try to adjust my PR furthermore.